### DISCUSSION TITLE
Complex multi lingual site setup - Advice needed

### DISCUSSION THREAD

**Discussion by Riaan-ZA** [statamic/cms#4846] [Q&A]
Hi, The setup I am looking to implement has the requirement of "regional" based content, meaning the ability to assign certain content to certain (geographic) regions. Initially I thought a multi site setup would work here however there is the additional requirement that each regions content should be available in multiple languages, which results in some sort of nested multi site setup which I don't think will work.

What would the best way to structure this sort of setup?

Ideally I would also like to cater for the following if possible:
 - Share common articles across regions (along with their translations)
 - Have editors limited to editing specific regions content

**Comment by edalzell** [comment]
> Have editors limited to editing specific regions content

This is not possible yet, there's an open FR for it.
